src/commands/login.ts

```diff
@@ -68,12 +47,39 @@ export default class LoginCommand extends Command {
     return success();
   }
 
+  private userNameQuestion(): any[] {
+    if (!this.token && !this.userName) {
+      return [{
+        name: "userName",
+        message:"Username or email: "
+      }];
+    }
+    return [];
+  }
+
+  private passwordQuestion(): any[] {
+    if (!this.token && !this.password) {
```

Comment:
Personally, I would prefer to move this logic to line 37 and only concat the password question if this condition is true. Feels a bit more natural, imo.